lib0/Cargo.toml

```diff
@@ -12,6 +12,7 @@ description = "Efficient binary encoding library for Yrs"
 lib0-serde = ["serde", "serde_json"]
 
 [dependencies]
+thiserror = "1"
```

Comment:
Added this dependency for convenient way to create typed errors through macro definitions.